src/ctree-app/ctree-landing.html

```diff
@@ -246,31 +376,55 @@
           <p>W</p>
           <p>W</p>
           <p>W</p>
-        </div><div id="participate" style="display: inline-block; width: 100vw; background-color: #FFFF00">
-          <section class="join">
-            <h2>Participate</h2>
-            <p>
-              <span class="bold">Join the open source project</span>
-              <br />
-              Get course credit or professional grade experience working on a
-              large scale open source project. Be part of a team, working
-              closely with other programmers and designers to complete a part of
-              a project. All regular contributors can add the project to their
-              LinkedIn resume and request a recommendation based on their
-              completed work. Go to
-              <span class="bold">dev.ctree.org</span> to get started.
-            </p>
-            <p>
-              <span class="bold">Test the project</span>
-              <br />
-              As the project develops we need real world examples of how it can
-              be used. If you want to use it with your nonprofit, online
-              community, or other organization please email
-              <span class="bold">contact@innovativeFuture</span>.
-            </p>
-          </section>
+          <p>W</p>
+          <p>W</p>
+          <p>W</p>
+        </div>
+        <div id="participate" class="section-styling">
+            <div class= "center-div">
+              <h1 align="center">Participate</h1>
+                <p>
+                  <div class="bold-text">Join the open source project</div>
```

Comment:
The text should all be centered, a bit smaller, and have less left/right padding for this section on mobile.  https://projects.invisionapp.com/share/HXN9A66V3ZE#/screens/311675692

Reply:
Done.

Reply:
This looks a little too small too.  As I said in the other comment, you should preview changes in the browser (I usually use 360 width).  You can compare how the text is laid out for the wireframe compared to the browser.

Reply:
Done.